packages/dialtone-css/lib/build/less/utilities/borders.less

```diff
@@ -22,11 +22,27 @@
 //  ============================================================================
 //  $$ DIRECTION
 //  ----------------------------------------------------------------------------
-.d-ba { border: var(--dt-size-100) solid !important; }
-.d-bt { border-top: var(--dt-size-100) solid !important; }
-.d-br { border-right: var(--dt-size-100) solid !important; }
-.d-bb { border-bottom: var(--dt-size-100) solid !important; }
-.d-bl { border-left: var(--dt-size-100) solid !important; }
+.d-ba {
+  border-color: var(--dt-color-border-default) !important;
+  border-style: solid !important;
+  border-width: var(--dt-size-border-100) !important;
+}
+
+.d-bt {
+  border-top: var(--dt-size-border-100) solid var(--dt-color-border-default) !important;
+}
+
+.d-br {
+  border-right: var(--dt-size-border-100) solid var(--dt-color-border-default) !important;
+}
+
+.d-bb {
+  border-bottom: var(--dt-size-border-100) solid var(--dt-color-border-default) !important;
+}
+
+.d-bl {
+  border-left: var(--dt-size-border-100) solid var(--dt-color-border-default) !important;
+}
```

Comment:
Shouldn't all of this properties be expanded too?

Reply:
Huh, could have sworn I did that. Anyway, done in 7dd0c94b51fc0af19a0b081ea85a418b1abb03bd